### PR TITLE
Feature/add quiet flag

### PR DESCRIPTION
**Pull request by rgugliel**
I'm calling Optimus quite a few time (dozen of times, actually) and its output (banner, constraints, etc) does not really suits me.

Here is a proposal for a quiet flag that limit the output.

### CODE REVIEW

**Comment by vagmcs** [comment]
Hello @rgugliel,

Yes we should do this change, but I think it should be made in a different way. In the develop branch I have added a logging library in the sbt dependencies, namely scala-logging, that is intended for this purpose. The quiet flag is not necessary, a logger should be used instead. Then if one does not want to see the output, should just close the logger.

Moreover, your branch "feature/addQuietFlag" I guess was created from the master branch not the develop branch. If you see in the develop branch the version is 2.1.1-SNAPSHOT not 2.1.0. 

To sum up, I will close this pull request, then either you can create a new pull request using the logger in a branch made from the develop branch or I can make the changes for you if you like.

**Comment by rgugliel** [comment]
Hi @vagmcs ,
Thanks for your answer!

Would moving the blocks I made optional with the quiet flag (banner, constraints, etc) to logger.info suit you? Then, one can change the verbosity of the logger.

Best,

Rafael

**Comment by vagmcs** [comment]
Yes @rgugliel, that should be ok. Remember to create a branch from develop, not master. :)

Vagelis
